### PR TITLE
Document that xds uses grpc-netty-shaded

### PR DESCRIPTION
**Pull request by lepistone**
For clarity, also extract grpc-netty-shaded to a separate paragraph.

Closes: #7869.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -55,10 +55,7 @@ Security.insertProviderAt(Conscrypt.newProvider(), 1);
 JDK versions prior to Java 9 do not support ALPN and are either missing AES GCM
 support or have 2% the performance of OpenSSL.
 
-We recommend most users use grpc-netty-shaded, which includes netty-tcnative on
-BoringSSL. It includes pre-built libraries for 64 bit Windows, OS X, and 64 bit
-Linux. For 32 bit Windows, Conscrypt is an option. For all other platforms, Java
-9+ is required.
+We recommend most users use [grpc-netty-shaded](#tls-with-grpc-netty-shaded).
```
Comment (sanjaypujare) [inline SECURITY.md:58]: How about rewording this to say something like "users of xDS management protocol" which is what the original issue was?

Comment (sanjaypujare) [reply SECURITY.md:58]: Waiting to address the comment so we can go ahead and merge this....

File: SECURITY.md [hunk 2]
```diff
@@ -55,10 +55,7 @@ Security.insertProviderAt(Conscrypt.newProvider(), 1);
 JDK versions prior to Java 9 do not support ALPN and are either missing AES GCM
 support or have 2% the performance of OpenSSL.
 
-We recommend most users use grpc-netty-shaded, which includes netty-tcnative on
```
Comment (ejona86) [inline SECURITY.md:58]: This paragraph shouldn't move; we need the Conscrypt and Java 9+ discussion here.

**Comment by sanjaypujare** [comment]
@lepistone could you address the comments so we can merge the PR? Thanks

**Comment by lepistone** [comment]
I removed the additional paragraph so I am now only adding a paragraph. This should address both comments. 

Sorry it took so long and thanks for reviewing!

**Review by sanjaypujare — APPROVED** [review]
LGTM
